CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.2
+
+_Sep 23, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇻🇳 Add Vietnamese (viVN) locale (#2668) @tuananh281098
+- 🇵🇱 Improve Polish (plPL) locale (#2632) @michallukowski
+- ⚡️ Apply the `valueFormatter` to `singleSelect` select options (#2581) @DanailH
+
+### `@mui/x-data-grid@v5.0.0-beta.2` / `@mui/x-data-grid-pro@v5.0.0-beta.2`
+
+#### Breaking changes
+
+- [DataGrid] The params passed to the `valueFormatter` were changed. (#2581) @DanailH
+  Use the `api` to get the missing params.
+  The `GridValueFormatterParams` interface has the following signature now:
+
+  ```diff
+  -export type GridValueFormatterParams = Omit<GridRenderCellParams, 'formattedValue' | 'isEditable'>;
+  +export interface GridValueFormatterParams {
+  +  /**
+  +   * The column field of the cell that triggered the event
+  +   */
+  +  field: string;
+  +  /**
+  +   * The cell value, but if the column has valueGetter, use getValue.
+  +   */
+  +  value: GridCellValue;
+  +  /**
+  +   * GridApi that let you manipulate the grid.
+  +   */
+  +  api: any;
+  +}
+  ```
```

Comment:
Would it make sense to revert this breaking change? It's a beta release, we have exited the alpha state, it didn't seem needed to fix #2514.